vpn/src/main/java/com/duckduckgo/mobile/android/vpn/processor/tcp/TCBCloser.kt

```diff
@@ -42,7 +42,7 @@ class TCBCloser @Inject constructor(private val socketWriter: TcpSocketWriter) {
         isFIN: Boolean,
     ) {
         val buffer = ByteBufferPool.acquire()
-        val tcb = requireNotNull(connectionParams.tcb())
+        val tcb = connectionParams.tcbOrClose() ?: return
```

Comment:
Added a bit of info in the description. TCBs can be null at times because we use an LRUCache and when we restart the VPN.
Although `requireNotNull` is OK, it'd crash the VPN processor forcing a restart. 
It's better approach to just free resources and continue to avoid unnecessary VPN process restarts